test/e2e/generator/generate.go

```diff
@@ -33,8 +33,18 @@ var (
 		"rocksdb":   10,
 		"cleveldb":  5,
 	}
-	nodeABCIProtocols    = uniformChoice{"unix", "tcp", "builtin", "grpc"}
-	nodePrivvalProtocols = uniformChoice{"file", "unix", "tcp", "grpc"}
+	nodeABCIProtocols = weightedChoice{
+		"builtin": 50,
+		"tcp":     20,
+		"grpc":    20,
+		"unix":    10,
+	}
+	nodePrivvalProtocols = weightedChoice{
+		"grpc": 35,
+		"tcp":  35,
+		"file": 20,
```

Comment:
I think in practice, file privVal's are most prevalent (I think it's also the default)

Reply:
perf, fixing.